execution/evm/vm_test.go

```diff
@@ -95,6 +95,476 @@ func TestVM(t *testing.T) {
 	}
 }
 
+func TestSHL(t *testing.T) {
+	ourVm := NewVM(newAppState(), newParams(), acm.ZeroAddress, nil, logger)
+	account1 := newAccount(1)
+	account2 := newAccount(1, 0, 1)
+
+	var gas uint64 = 100000
+
+	//Shift left 0
+	bytecode := MustSplice(PUSH1, 0x01, PUSH1, 0x00, SHL, return1())
+	output, err := ourVm.Call(account1, account2, bytecode, []byte{}, 0, &gas)
+	value := []uint8([]byte{0x1})
+	expected := LeftPadBytes(value, 32)
+	assert.Equal(t, expected, output)
+
+	fmt.Printf("Result: %v == %v\n", output, expected)
```

Comment:
I know we use `fmt.Printf` elsewhere but it would be preferable to use `t.Logf` that give more metadata and can be redirected to other writers.